addons/resource.language.en_gb/resources/strings.po

```diff
@@ -19425,7 +19429,7 @@ msgstr ""
 #. Description of setting with label #791 "Allow remote control from programs on this system"
 #: system/settings/settings.xml
 msgctxt "#36334"
-msgid "Allow programs on this device to control this application via the web interface or the JSON-RPC interface protocol."
+msgid "Allow programs on this device to control this application via the JSON-RPC or EventServer protocol, without authentication."
```

Comment:
Is "JSON-RPC" clear enough? It only controls two of [the four possible transports](https://kodi.wiki/view/JSON-RPC_API#Transports) - WS and TCP.

Reply:
I just tried to make it clearer than before, but that does not mean it cannot be improved. What would you suggest?

Reply:
That is a tough one. Maybe "JSON-RPC socket or EventServer protocol" could do it? TCP is noted as a "socket" in the [wiki page on JSON-RPC](https://kodi.wiki/view/JSON-RPC_API#TCP). The word may not be too meaningful to most users, but it is a clear term to search for.